dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -283,6 +283,8 @@ class BGPPeerMgr(Manager):
         }
 
     def set_handler(self, key, data):
+        if not self.directory.path_exist("meta", "localhost/bgp_asn"):
+            return False
```

Comment:
I think that the change must be following:
```
     def on_deps_change(self):
+        if not self.directory.available_deps(self.deps):
+            return
         new_queue = []
         for key, data in self.set_queue:
             res = self.set_handler(key, data)
```
I'll have it on my new PR I'm currently working.

Reply:
> I'd suggest to close your PR and merge #4027

OK, thanks!